src/classifier/Features.h

```diff
@@ -24,6 +24,11 @@
 #ifdef _WIN32
 #include <stdint.h>
 #endif
+#include <string>
+#include <unordered_map>
```

Comment:
Did you make sure this compiles in linux using the provided makefile? I don't think it will, since unordered_map is in tr1. I'm updating the master branch to use the flag -std=c++0x which will solve this issue, but in any case, please make sure the code runs in linux before requesting a pull.